packages/kit/src/runtime/paths.spec.js

```diff
@@ -0,0 +1,14 @@
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+import { set_paths, base, assets } from './paths.js';
+
+test('paths', () => {
```

Comment:
`set_paths` might be a clearer name

but I question if this is really a useful test. just testing that setting a variable sets it - there doesn't seem to be much room for error. and I would expect this is already covered by our other tests

Reply:
removed